app/src/main/java/com/my/journeyplanner/helpers/JourneyPlannerApiService.kt

```diff
@@ -13,16 +13,21 @@ interface JourneyPlannerApiService {
     fun getJourneyResults(
         @Path("fromLocation") fromLocation: String,
         @Path("toLocation") toLocation: String
-    ): Call<JourneyPlannerDisambiguationResult>
+    ): Call<JourneyPlanner.ItineraryResult>
 
     companion object {
         private const val TFL_API_BASE_URL = "https://api.tfl.gov.uk/"
-        private val client = OkHttpClient().newBuilder().addInterceptor(ParametersInterceptor()).build()
+        private val client = OkHttpClient().newBuilder()
+            .addInterceptor(ParametersInterceptor())
+            .build()
 
-        fun createApiService(): JourneyPlannerApiService = Retrofit.Builder()
+        val retrofit: Retrofit = Retrofit.Builder()
```

Comment:
is this change necessary? what have we improved here? Specially concerned about making it public

Reply:
Used to access JourneyPlannerApiService.retrofit.responseBodyConverter().
I can use fakeRetrofit:
        val fakeRetrofit: Retrofit = Retrofit.Builder()
            .baseUrl("https://ttttt.com/")
            .client(OkHttpClient().newBuilder().build())
            .addConverterFactory(MoshiConverterFactory.create())
            .build()
Am I missing the point?

Reply:
Renamed to MockRetrofit using MockInterceptor() for responseBodyConverter(). I'd moved this into a separate class but I suspect you have a better solution!

Reply:
Removed as no longer required